includes/User.php

```diff
@@ -246,6 +247,11 @@ class User {
 	static $idCacheByName = array();
 
 	/**
+		* @var string the service auth token (currently helios); should NEVER be cached
+	 */
+	private $globalAuthToken = null;
```

Comment:
Yup. It was [caching the whole user](https://github.com/Wikia/app/pull/8628/files#diff-fd654af4c113963df43fbd159b1569c8L257).